mixins/kubernetes/alerts/apps_alerts.libsonnet

```diff
@@ -33,7 +33,7 @@
             expr: |||
               sum by (namespace, pod, %(clusterLabel)s) (
                 max by(namespace, pod, %(clusterLabel)s) (
```

Comment:
@vishiy - Do we need to make corresponding changes in our recommended alerts too after this is merged if anything relevant has changed?

Reply:
As discussed offline, these are not used currently and we have modifications